src/components/Header/HeaderMenu.tsx

```diff
@@ -1,21 +1,31 @@
+import { useAppStore } from "@/hooks/useAppStore";
+import { useConfigurationStore } from "@/stores/configurationStore";
 import Link from "next/link";
 import { usePathname } from "next/navigation";
-import { useConfigurationStore } from "@/stores/configurationStore";
+import { ToggleSwitch } from "../Common/ToggleSwitch";
 
 export default function HeaderMenu() {
+    const { favouriteStations } = useAppStore();
     const pathname = usePathname();
     const { menu } = useConfigurationStore();
-    
-    return (<section>
-        {menu.map(element => 
-            <Link
-                className={`px-2 text-lg text-primary ${pathname === element.pathName ? "font-bold text-success" : ""}`}
-                href={element.pathName}
-                key={element.text}
-            >
-                {element.text}
-            </Link>
-        )}
-    </section>
+
+    return (
+        <section className="flex flex-col">
+            <div className="mt-6 flex items-center gap-4 lg:gap-0">
+                {menu.map((element) => (
+                    <Link
+                        className={`px-2 text-lg text-primary ${
+                            pathname === element.pathName ? "font-bold text-success" : ""
+                        }`}
+                        href={element.pathName}
+                        key={element.text}
+                    >
+                        {element.text}
+                    </Link>
+                ))}
+            </div>
+
+            {favouriteStations.length > 0 && <ToggleSwitch label={"My Favourite Stations"} />}
```

Comment:
This addition here is my main objection to this project. I do feel it is too much to have this in the header as an option. 
The header is common for all the pages at the moment as a result the switch will be also visible in the warnings page (and in the future about us page or any other irrelevant one). Maybe it would be better to move it to the necessary views only?

Reply:
On the sage topic, I was thinking about adding options like windy does on the right of the map screen. That space is occupied by warnings at the moment (desktop) which should be revamped, maybe we could use the favorite station feature as the breaking ground for this. Or maybe, we could merge this (not push it to prod just yet) while I revamp the warnings feature.